space_robots/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+# Setup the Demo environment
+source "${SPACEROS_DIR}/install/setup.bash"
```

Comment:
Shouldn't this entrypoint script the `DEMO_DIR` install folder instead of the base `SPACEROS_DIR`?